mineral-ui/mineral-ui
# [Link] Fix console warning.  Don't require children.

### Description

* Fix console warning for variant prop
* Do not require children
* Update glamorous dependency

### Motivation and context

closes #317 
closes #255 

### How to test

1. `npm i`
2. `npm start`
3. Navigate to Link section
4. open devTools
5. note that there are no console warnings for the Link component

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### Checklist
